tests/cypress/support/commands.ts

```diff
@@ -122,7 +122,19 @@ Cypress.Commands.add('verifyTableRow', (rowNumber, expectedText1, expectedText2)
   cy.get(`table > tbody > tr.main-row[data-testid="sortable-table-${rowNumber}-row"]`)
     .children({ timeout: 60000 })
     .should('contain', expectedText1 )
-    .should('contain', expectedText2 ); // TODO: refactor this so it is not mandatory value
+    // Check if expectedText2 is a RegExp or a string
+    .then(() => {
+      if (expectedText2 instanceof RegExp) {
```

Comment:
Just thinking loud / things to consider:
- This assumes that the `expectedText2` must be used for cluster counting. Intially it was thought that cluster counting to be either in `expectedText1` or `expectedText2`. 
- Perhaps could we set an env var to use this regexp if it detects is ci?
- Perhaps better to use a flag to indicate if app to be deployed in 1 ds or more and which one specifically?

Let's pls discuss separately to agree on a solution

Reply:
This was the simplest solution I could think of, I agree it is not optimal.

Reply:
And I think is a nice one; don't get me wrong.
Let's just think if in terms of building the test we want this as default and later add a flag to specifically target the selected ds or not. Also, we can think if this can be somehow called solely when ci uses it or just leave it like that for the moment.

Reply:
IMO, this is extra check for single element i.e. 1/1 or 0/0. But the usage of verifyTableRow is not limited to that, we have another entries to check there as well.

This is not blocking as well, just my thought :smile: